lib/profiles.js

```diff
@@ -22,7 +22,11 @@ const KMS_MODULE = 'ssm-v1';
  */
 exports.create = async ({accountId} = {}) => {
   assert.string(accountId, 'accountId');
-  const {profileAgent} = await profileAgents.create({accountId});
+  // FIXME: if profileId needs to be a required param in the create API,
+  // then a placeholder must be used here until the profile is updated
+  // to a did:key later in this function
```

Comment:
@gannan08 @dlongley , a new FIXME here.

Reply:
I see, we don't know the profile ID until after the agent is created. In that case, it should not be required.